daemon/protocol/python/scoot/client_lib.py

```diff
@@ -90,13 +93,53 @@ def start():
     raise ScootException(Exception("Could not find domain socket path."))
   if is_started():
     raise ScootException(Exception("Already started."))
-  channel = grpc.insecure_channel("unix://%s" % _domain_sock)
+  channel = None
+  try:
+    channel = grpc.insecure_channel("unix://%s" % _domain_sock)
+  except Exception as e:
+    raise ScootException("Error getting channel:'unix://{}'".format(_domain_sock))
   #channel.subscribe(lambda x: sys.stdout.write(str(x)+"\n"), try_to_connect=True)
   _client = daemon_pb2.ScootDaemonStub(channel)
+  
+  # verify the connection
+  echoReturn = None
```

Comment:
Looking for UNAVAILABLE in echoReturn is kind of weird. Why not keep the error around and check for non-null errors explicitly?